pkgs/development/python-modules/torchvision/default.nix

```diff
@@ -49,14 +49,30 @@ buildPythonPackage {
     owner = "pytorch";
     repo = "vision";
     rev = "refs/tags/v${version}";
-    hash = "sha256-KNbOgd6PCINZqZ24c/Ev+ODux3ik5iUlzem9uUfQArM=";
+    hash = "sha256-Rqrh9XnRIJYaYwKsZEHo4YgCLD4k/U7MzB4jDk1rSyI=";
   };
 
-  nativeBuildInputs = [ libpng ninja which ] ++ lib.optionals cudaSupport [ cuda-native-redist ];
+  nativeBuildInputs = [
+    libpng
+    ninja
+    which
+  ] ++ lib.optionals cudaSupport [
+    cuda-native-redist
+  ];
 
-  buildInputs = [ libjpeg_turbo libpng ] ++ lib.optionals cudaSupport [ cuda-redist ];
+  buildInputs = [
+    libjpeg_turbo
+    libpng
+  ] ++ lib.optionals cudaSupport [
+    cuda-redist
+  ];
 
-  propagatedBuildInputs = [ numpy pillow torch scipy ];
+  propagatedBuildInputs = [
+    numpy
+    pillow
+    torch
+    scipy
+  ];
```

Comment:
what's the relevant change here?

Reply:
Nothing, just formatting.
Do you want me to revert them ?

Reply:
yeah i say let's prefer to keep the git blame clean